OpenSim/Simulation/Model/Appearance.h

```diff
@@ -33,105 +33,157 @@ namespace OpenSim {
 class Body;
 class Model;
 
-
-//=============================================================================
-//=============================================================================
+using SimTK::DecorativeGeometry;
```

Comment:
It is generally a bad idea to put `using` statements in header files because they quietly introduce symbols into the namespaces of anyone who includes the header, even indirectly. You may be able to put the `using` statement within a class declaration; if not you should write it out explicitly with the `SimTK::` prefix. You can of course put `using` at the top of any `.cpp` file so that the implementation does not have to be cluttered the same way as the header.